packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -226,6 +226,15 @@ class DescopeWc extends BaseDescopeWc {
     const loginId = this.sdk.getLastUserLoginId();
     const flowConfig = await this.getFlowConfig();
     const projectConfig = await this.getProjectConfig();
+    const flowVersions = {
+      version: flowConfig.version,
+      componentsVersion: projectConfig.componentsVersion,
+      versions: Object.keys(projectConfig.flows || []).reduce((acc, key) => {
```

Comment:
why do we need other flows version here?

Reply:
When starting flow with subflows. we need to know the version for all participate flows.

Reply:
cool, maybe we should add something like `// pass other flow versions, that may be used as a part of the current flow`